lib/facility.rb

```diff
@@ -1,14 +1,88 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name, 
+              :address, 
+              :phone, 
+              :services,
+              :registered_vehicles,
+              :collected_fees
 
-  def initialize(name, address, phone)
+  def initialize(name:, 
```

Comment:
You're using here, what's called keyword arguments. I really like that! Keeps it very clear what each argument is.